static/_redirects

```diff
@@ -365,8 +367,8 @@
 /docs/user-guide/monitoring/     /docs/tasks/debug-application-cluster/resource-usage-monitoring/ 301
 /docs/user-guide/namespaces/     /docs/concepts/overview/working-with-objects/namespaces/ 301
 /docs/user-guide/networkpolicies/     /docs/concepts/services-networking/network-policies/ 301
-/docs/user-guide/node-selection/     /docs/concepts/configuration/assign-pod-node/ 301
-/docs/user-guide/node-selection/README     /docs/concepts/configuration/assign-pod-node/ 301
+/docs/user-guide/node-selection/     /docs/concepts/scheduling-eviction/assign-pod-node/ 301
+/docs/user-guide/node-selection/README     /docs/concepts/scheduling-eviction/assign-pod-node/ 301
```

Comment:
@jimangel Missed these on the first pass - now fixed (perhaps the double-redirect could have been an issue?).